hw3/exercise-3/src/components/Search.js

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from "react";
+//import {FaSearch} from "react-icons/fa"
+import "./search.css";
+
+const Search = () => {
+	const [character, setCharacter] = useState("");
+	const [selected, setSelected] = useState(null);
+
+	const url = "https://thronesapi.com/api/v2/Characters";
+	const fetchData = (value) => {
+		fetch(url)
+			.then((response) => {
+				return response.json();
+			})
+			.then((data) => {
+				const results = data.find((character) => {
```

Comment:
how do you handle the case when someone enters 'stark' and there is more than one match?

Reply:
or someone enters just a single ' ' character?